js/src/api/contract/contract.js

```diff
@@ -232,51 +233,102 @@ export default class Contract {
       return this._subscribe(event, options, callback);
     };
 
+    event.register = (options = {}, callback) => {
```

Comment:
I guess for filters querying pending blocks too it could be useful to query every second (not sure though). True from the API perspective, but didn't want to change the whole behavior.